apps/webapp/app/components/navigation/SideMenu.tsx

```diff
@@ -4,7 +4,6 @@ import {
   ArrowRightOnRectangleIcon,
```

Comment:
The changes in this file are improvements to the sidebar, mostly unrelated to deleting/renaming projects.
- The dropdown title is now the Project name (instead of the org name)
- The section titles are now "Project" and "Organization" because it was confusing before.